src/gui/transferlistmodel.cpp

```diff
@@ -55,6 +55,76 @@ static QIcon getErrorIcon();
 
 static bool isDarkTheme();
 
+TransferListTextColors::TransferListTextColors()
+    : m_unknown {getColorByState(BitTorrent::TorrentState::Unknown)}
+    , m_forcedDownloading {getColorByState(BitTorrent::TorrentState::ForcedDownloading)}
+    , m_downloading {getColorByState(BitTorrent::TorrentState::Downloading)}
+    , m_downloadingMetadata {getColorByState(BitTorrent::TorrentState::DownloadingMetadata)}
+    , m_allocating {getColorByState(BitTorrent::TorrentState::Allocating)}
+    , m_stalledDownloading {getColorByState(BitTorrent::TorrentState::StalledDownloading)}
+    , m_forcedUploading {getColorByState(BitTorrent::TorrentState::ForcedUploading)}
+    , m_uploading {getColorByState(BitTorrent::TorrentState::Uploading)}
+    , m_stalledUploading {getColorByState(BitTorrent::TorrentState::StalledUploading)}
+    , m_checkingResumeData {getColorByState(BitTorrent::TorrentState::CheckingResumeData)}
+    , m_queuedDownloading {getColorByState(BitTorrent::TorrentState::QueuedDownloading)}
+    , m_queuedUploading {getColorByState(BitTorrent::TorrentState::QueuedUploading)}
+    , m_checkingUploading {getColorByState(BitTorrent::TorrentState::CheckingUploading)}
+    , m_checkingDownloading {getColorByState(BitTorrent::TorrentState::CheckingDownloading)}
+    , m_pausedDownloading {getColorByState(BitTorrent::TorrentState::PausedDownloading)}
+    , m_pausedUploading {getColorByState(BitTorrent::TorrentState::PausedUploading)}
+    , m_moving {getColorByState(BitTorrent::TorrentState::Moving)}
+    , m_missingFiles {getColorByState(BitTorrent::TorrentState::MissingFiles)}
+    , m_error {getColorByState(BitTorrent::TorrentState::Error)}
+{
+}
+
+QColor TransferListTextColors::textColorByState(BitTorrent::TorrentState state) const
```

Comment:
`const BitTorrent::TorrentState state`

Reply:
sorry for such a stupid mistake again
done.